cmd/panfs-multipart.go

```diff
@@ -876,12 +880,6 @@ func (fs *PANFSObjects) AbortMultipartUpload(ctx context.Context, bucket, object
 	delete(fs.appendFileMap, uploadID)
```

Comment:
may be defer defete would be better?

Reply:
`fs.appendFileMap` is used under the lock here so do we need need to use `defer` in that case? That is just a map and we need to remove a record which indicate that existing multipart upload file is not longer valid